charts/sonarqube-dce/values.schema.json

```diff
@@ -1,6 +1,11 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
     "properties": {
+        "nginx.enabled": {
```

Comment:
I think we should deprecate nested values as we did for example for "readinessProbe.sonarWebContext"